src/types.ts

```diff
@@ -32,8 +34,22 @@ export interface ZAuctionInstances {
   [registrarAddress: string]: zAuction.Instance;
 }
 
-export interface RouteUriToInstance {
-  [key: string]: ZAuctionInstances;
+export interface NetworksToAddresses {
```

Comment:
This interface is called `NetworksToAddresses` but in the definition you are mapping `address` to `AddressesToTokenApiInfo`. I'm assuming you meant `[network: string]` ?

Also what is this mapping really of? What are these addresses? Think of `address` as a raw type like `number` or `string`. I don't think `NetworksToNumber` would be intuitive to a user.